cmd/antrea-agent/agent.go

```diff
@@ -94,7 +94,8 @@ func run(o *Options) error {
 	networkConfig := &config.NetworkConfig{
 		TunnelType:        ovsconfig.TunnelType(o.config.TunnelType),
 		TrafficEncapMode:  encapMode,
-		EnableIPSecTunnel: o.config.EnableIPSecTunnel}
+		EnableIPSecTunnel: o.config.EnableIPSecTunnel,
+		DefaultMTU:        o.config.DefaultMTU}
```

Comment:
I think NetworkConfig.DefaultMTU is used nowhere except passing the value to agentInitializer? If that is case, I would not add it to NetworkConfig, but still keep it an argument of NewInitializer().
NetworkConfig is for keeping some network parameters used by multiple files/packages.